chain/jsonrpc/src/lib.rs

```diff
@@ -38,10 +38,11 @@ use near_primitives::rpc::{
 use near_primitives::serialize::{from_base, from_base64, BaseEncode};
 use near_primitives::transaction::SignedTransaction;
 use near_primitives::types::{AccountId, BlockId, BlockReference, MaybeBlockId};
-use near_primitives::utils::is_valid_account_id;
 use near_primitives::views::{
     FinalExecutionOutcomeView, FinalExecutionOutcomeViewEnum, QueryRequest,
 };
+use near_runtime_utils::is_valid_account_id;
```

Comment:
it doesn't work without moving this function because the internals of runtime can't depend on near_primitives. 
This can be split into separate PR if someone has time though